### DISCUSSION TITLE
Modal popup with NavigateTo does not load the next page properly, the vertical scroll is missing

### DISCUSSION THREAD

**Discussion by arcdotnet** [vikramlearning/blazorbootstrap#208] [Q&A]
I tried to use the Blazor Bootstrap's Modal, but there is a problem when putting **NavigateTo** on the button inside the modal, it will show the next page, but the vertical scroll is missing.

`<Modal @ref="modal" Title="Modal title">
    <BodyTemplate>
        Modal body text goes here.
    </BodyTemplate>
    <FooterTemplate>
        <Button Color="ButtonColor.Secondary" @onclick="OnHideModalClick">Close</Button>
        <Button Color="ButtonColor.Primary">Save changes</Button>
    </FooterTemplate>
</Modal>

<Button Color="ButtonColor.Primary" @onclick="GoToNextPage">Show Modal</Button>

@code {
    private Modal modal = default!;

    private async Task GoToNextPage()
    {
        Navman.NavigateTo("/MyPage2");
    }

    private async Task OnHideModalClick()
    {
        await modal.HideAsync();
    }
}`

I hope you can help me on this.

Thanks

**Comment by gvreddy04** [comment]
@arcdotnet Thank you for trying BlazorBootstrap components.

Please share a sample GitHub repo to reproduce the issue. Also, share the below details:

- .NET Version: 
- BlazorBootstrap Version: 
- WebAssembly / Server: 
- Screenshot:
